declarations.d.ts

```diff
@@ -559,3 +560,106 @@ interface IPluginVariablesHelper {
 	getPluginVariableFromVarOption(variableName: string, configuration?: string): any;
 	simplifyYargsObject(obj: any, configuration?: string): any;
 }
+
+/**
+ * Describes Registry values are returned from winreg
+ */
+interface IWinRegResult {
```

Comment:
this should go into a separate .d.ts, potentially donating it to DefinitelyTyped